data/config-schema/src/main/java/net/corda/schema/configuration/StateManagerConfig.java

```diff
@@ -1,15 +1,25 @@
 package net.corda.schema.configuration;
 
 /**
- * Configuration keys to access public parts of the configuration under the {@code corda.stateManager} key.
+ * Configuration keys for State Manager config used to bootstrap a worker.
  */
 public final class StateManagerConfig {
     private StateManagerConfig() {
     }
 
     public static final String STATE_MANAGER = "stateManager";
-
     public static final String TYPE = "type";
+    public static final String STATE_TYPE = "stateType";
+
+    // Keys for state type configuration
+    public static final class StateType {
+        public static final String DEFAULT = "default";
```

Comment:
I'm not sure that adding a `default` state type is a good idea... can we remove it and, instead, enforce the existence of a real type when deploying Corda through the helm chart?.